Jenkinsfile-e2e

```diff
@@ -53,7 +53,8 @@ node {
               ]
 
               if (env.BRANCH_NAME == 'latest') {
-                notifySlack(messageParameters)
+                notifySlack(messageParameters, '#si_repo-simorgh')
```

Comment:
Could these slack channel names be moved to a var at the top of the file and only referenced here

Reply:
will make it much easier to update channel names if we ever need to.

Reply:
yes will look into that